src/aragonjs-wrapper.js

```diff
@@ -234,21 +234,22 @@ const subscribe = (
 const initWrapper = async (
   dao,
   {
-    provider,
-    walletProvider = null,
+    guiStyle = null,
```

Comment:
We should add `guiStyle` to the initialization options of `new Aragon()`

Reply:
Ah yes totally!

Reply:
Ah I am checking again, and we already have it on the `init()` options. Would you remove the option from `init()`?

Reply:
Ahhh I totally misread the PR on aragon.js and thought it was on the constructor and not the init. You're right, this is all good!